src/main/java/net/mcreator/ui/modgui/ProcedureGUI.java

```diff
@@ -292,6 +296,13 @@ public Component getListCellRendererComponent(JList<? extends VariableElement> l
 	}
 
 	@Override protected void initGUI() {
+		skipDependencyNullCheck.setOpaque(false);

```

Comment:
To use less space, I would put this on the bottom, similar to "GUI procedure triggers" in GUI editor and same way make it collapsible. This also makes sure users won't just blindly check thi